docs/interfaces/meetingsession.html

```diff
@@ -91,7 +91,7 @@ <h3>Properties</h3>
 								<li class="tsd-kind-property tsd-parent-kind-interface"><a href="meetingsession.html#configuration" class="tsd-kind-icon">configuration</a></li>
 								<li class="tsd-kind-property tsd-parent-kind-interface"><a href="meetingsession.html#contentshare" class="tsd-kind-icon">content<wbr>Share</a></li>
 								<li class="tsd-kind-property tsd-parent-kind-interface"><a href="meetingsession.html#devicecontroller" class="tsd-kind-icon">device<wbr>Controller</a></li>
-								<li class="tsd-kind-property tsd-parent-kind-interface"><a href="meetingsession.html#eventreporter" class="tsd-kind-icon">event<wbr>Reporter</a></li>
```

Comment:
@ltrung totally random but i just wanted to check....is there any other way to do docs that doesn't require committing the regenerated along with every commit? :P